projects/aca-playwright-shared/src/page-objects/components/manageRules/manage-rules.component.ts

```diff
@@ -31,8 +31,42 @@ export class ManageRules extends BaseComponent {
   public getGroupsList = (optionName: string): Locator => this.getChild('.aca-rule-list-item__header', { hasText: optionName });
   public ruleToggle = this.getChild('.aca-manage-rules__container .mat-slide-toggle-bar').first();
   public ruleToggleFalse = this.getChild('aca-rule-list-grouping input[type="checkbox"][aria-checked="false"]').first();
+  public ruleDetailsTitle = this.getChild('.aca-manage-rules__container__rule-details__header__title__name');
+  public ruleDetailsTrashButton = this.getChild('#delete-rule-btn');
+  public ruleDetailsEditButton = this.getChild('#edit-rule-btn');
+  public ruleDetailsWhenText = this.getChild('[data-automation-id="rule-details-triggers-component"]');
+  public ruleDetailsPerformActionsDiv = this.getChild('adf-card-view-textitem mat-form-field input');
+  public rulesEmptyListIcon = this.getChild('mat-icon', { hasText: 'library_books' });
```

Comment:
can we check the empty list by its tag, instead of the content? the icon is just a content of the empty list component I think

Reply:
Yes I think we can do that. I will fix that in the next push